pavelib/utils/test/bokchoy_utils.py

```diff
@@ -77,6 +77,9 @@ def wait_for_server(server, port):
         )
     )
 
+    if tasks.environment.dry_run:
```

Comment:
It's built in to paver. Not sure there's much use to it except debugging paver stuff, honestly.